author-guide.qmd

```diff
@@ -68,6 +68,15 @@ JOVI encourages you to use structured abstracts, with distinct, labeled sections
 | Materials       |                 |
 | Conclusion      |                 |
 
+#### References
+
+For any reference entry that has a DOI, please append the DOI URL (prefixed with https://doi.org/) at the end of the reference entry. For example, see the last part of the following citation:
+
+> Besançon, Lonni, Anastasia Bezerianos, Pierre Dragicevic, Petra Isenberg, and Yvonne Jansen. 2021. “Publishing Visualization Studies as Registered Reports: Expected Benefits and Researchers’ Attitudes.” https://doi.org/10.31219/osf.io/3z7kx.
+
+This DOI URL allows CrossRef to recognize the references correctly. 
+This issue is essential because some institutions still use citation counts from CrossRef for hiring or promotion.
```

Comment:
Not sure we need this last statement justifying institutions' bean counting hiring policies? heh

Reply:
This sentence was my idea. I want to add a reason why it is important to have correct CrossRef records, and this is what came up in my head. 

If any of you have a better reason, feel free to replace.